docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -240,6 +240,14 @@ There you can find links to upgrade notes for other versions too.
         - $container->setParameter('shopsys.router.locale_router_filepaths', $config['router']['locale_router_filepaths']);
         + $container->setParameter('shopsys.router.locale_router_filepath_mask', $config['router']['locale_router_filepath_mask']);
         ```
+- add the installation of the useful tools to your `docker/php-fpm/Dockerfile` ([#1239](https://github.com/shopsys/shopsys/pull/1239))
+    ```diff
+    libpq-dev \
```

Comment:
`libpq-dev` already was there in Dockerfile... Or I don't understand this line in the diff.

Reply:
It's about context – where to add tools. From output it's more clear 
![Snímek obrazovky 2019-07-23 v 12 37 54](https://user-images.githubusercontent.com/1177414/61705819-ddae1480-ad46-11e9-9cbb-f380a4f98f91.png)

Reply:
Yay... my fault... I didn't see missing `+` on this line... OK :)